pkgs/development/compilers/idris2/default.nix

```diff
@@ -3,9 +3,14 @@
 }:
 
 # Uses scheme to bootstrap the build of idris2
-stdenv.mkDerivation rec {
-  name = "idris2";
-  version = "0.2.1";
+let
+  idris2name = "idris2";
+  idris2version = "0.2.1";
+  includedLibs = [ "base" "contrib" "network" "prelude" ];
```

Comment:
Purely cosmetic, but can't this be just `pname = "idris2"; version = "0.2.1"; name = "${pname}-${version}" rather than defining `idris2name`, etc.?

Reply:
Similarly, why do we need to lift `includedLibs` up to the top here rather than binding them within `postInstall`?

Reply:
I wanted to have `includedLibs` separate as I expect to create packages from them later. But you're absolutely correct, neither the idris2version construct not the includedLibs are strictly necessary. These are precursors to potential build arguments. I don't expect these to stay that way, so I'm fine with it for now.